images/kubekins-e2e/Dockerfile

```diff
@@ -96,7 +96,7 @@ RUN if [ -n "${KIND_VERSION}" ]; then \
 # kubetest2 must be installed from git as the makefile embeds the git short hash and the day it was built.
 ARG KUBETEST2_VERSION
 RUN if [ -n "${KUBETEST2_VERSION}" ]; then \
-    git clone --depth=1 --branch ${KUBETEST2_VERSION}  https://github.com/kubernetes-sigs/kubetest2.git  /tmp/kubetest2 && \
+    git clone https://github.com/kubernetes-sigs/kubetest2.git /tmp/kubetest2 && \
```

Comment:
`--filter=tree:0` gives good results without the tag data issue of shallow clones (e.g. here https://github.com/kubernetes-sigs/kind/blob/80a64d961b88c31e1812f5ba57d689bbba398c27/images/base/Dockerfile#L139)

not a hard requirement though
/lgtd
/approve
/hold